### PR TITLE
chore(README): update link to Discord.js guide v12 changes

### PR DESCRIPTION
**Pull request by PLASMAchicken**
**Please describe the changes this PR makes and why it should be merged:**
The v12-changes of the guide has been merged into master, so the old link is invalid now.
This fixes this issue.

**Status**
- [x] Code changes have been tested against the Discord API, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ client.login('token');
 * [Website](https://discord.js.org/) ([source](https://github.com/discordjs/website))
 * [Documentation](https://discord.js.org/#/docs/main/master/general/welcome)
 * [Guide](https://discordjs.guide/) ([source](https://github.com/discordjs/guide)) - this is still for stable  
-  See also the WIP [Update Guide](https://github.com/discordjs/guide/blob/v12-changes/guide/additional-info/changes-in-v12.md) also including updated and removed items in the library.
+  See also the WIP [Update Guide](https://github.com/discordjs/guide/blob/master/guide/additional-info/changes-in-v12.md) also including updated and removed items in the library.
```
Comment (RDambrosio016) [inline README.md]: Might as well remove the `WIP` too, i also dont see why you dont just link to the actual guide which looks better https://discordjs.guide/additional-info/changes-in-v12.html

Comment (tipakA) [reply README.md]: Not only looks better, but actually works. All links like those:
image
do not link to github repo, but are internal links for guide's website itself, meaning that if you click on them on github, they don't work at all.

Comment (RDambrosio016) [reply README.md]: True, forgot about that too 👍

File: README.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ client.login('token');
 * [Website](https://discord.js.org/) ([source](https://github.com/discordjs/website))
 * [Documentation](https://discord.js.org/#/docs/main/master/general/welcome)
 * [Guide](https://discordjs.guide/) ([source](https://github.com/discordjs/guide)) - this is still for stable  
-  See also the WIP [Update Guide](https://github.com/discordjs/guide/blob/v12-changes/guide/additional-info/changes-in-v12.md) also including updated and removed items in the library.
+  See also the [Update Guide](https://discordjs.guide/additional-info/changes-in-v12.html) also including updated and removed items in the library.
```
Comment (amishshah) [inline README.md]:
```suggestion
  See also the [Update Guide](https://discordjs.guide/additional-info/changes-in-v12.html), including updated and removed items in the library.
```

The second also is redundant

**Pull request merged**